docs/guides/helpers.rst

```diff
@@ -48,4 +48,4 @@ Interface and annotations
 
 .. code:: php
 
-   elgg_view_entity_icon($user, 'small', array('use_hover' => false));
\ No newline at end of file
+   elgg_view_entity_icon($user, 'small', array('use_hover' => false));
```

Comment:
-- ``elgg_get_loggedin_user_entity()`` Returns the ElggUser for the current user
-- ``elgg_get_loggedin_user_guid()`` Returns the GUID of the current user
+- ``elgg_get_logged_in_user_entity()`` Returns the ElggUser for the current user
+- ``elgg_get_logged_in_user_guid()`` Returns the GUID of the current user